src/ansys/dpf/core/data_tree.py

```diff
@@ -482,6 +482,91 @@ def get_as(self, name, type_to_return=types.string):
             out = DataTree(data_tree=obj, server=self._server)
         return out
 
+    def get_attribute_names(self):
+        """
+        Returns a list of defined attribute names.
+
+        Returns
+        -------
+        list[str]
+
+        Examples
+        --------
+        >>> from ansys.dpf import core as dpf
+        >>> data_tree = dpf.DataTree()
+        >>> data_tree.add(id=3, qualities=["nice", "funny"], name="George")
+        >>> data_tree.get_attribute_names()
+        ['id', 'name', 'qualities']
+        """
+        coll_obj = collection.StringCollection(
+            collection=self._api.dpf_data_tree_get_available_attributes_names_in_string_collection(
+                self
+            ),
+            server=self._server,
+        )
+
+        return coll_obj.get_integral_entries()
+
+    def get_sub_tree_names(self):
```

Comment:
Could we use properties here please to be consistent with other classes, ie: "attribute_name", "sub_stree_names"